File: examples/python/micro/user_service.py
from .auth_service import verify_token, run_admin_command


def get_user(token):
    user_id = verify_token(token)
    return {"id": user_id, "token": token}


def update_user(token, action):
    user_id = verify_token(token)
    if user_id:
        run_admin_command(user_id, action)
    return user_id

File: examples/elixir/micro/user_service.ex
defmodule MyApp.UserService do
  alias MyApp.AuthService

  def get_user(token) do
    AuthService.verify_token(token)
  end

  def update_user(token, action) do
    user_id = AuthService.verify_token(token)

    if user_id do
      AuthService.run_admin_command(user_id, action)
    end

    user_id
  end
end
